1zadanie.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <link rel="stylesheet" href="1zadanie.css" type="text/css">
+    <link href="https://fonts.googleapis.com/css?family=Lato:200,300,300i,400,400i,700,700i&amp;subset=latin-ext" rel="stylesheet">
+    
+    </head>
+<body>
+
+    
+     <section id="layout" >
+         <img src="coming-soon-ribbon.png" id="coming_soon">
+         <img src="undraw_developer_activity_bv83.svg" id="guy" >
+         <div id="code" style="color: white;">CODE ACADEMY</div>
+         <h3>Leave your email</h3>
+    <div id="get_mail">
+        <div class="form-group">
+        <div class="sexiflexi"> <input required type="email" name="email" id="email_field" class="form-control">
+        <label for="email" class="form-control-placeholder" width="30%">E-mail</label>
+            </div>
+            </div>
+        <div class="sexiflexi"> <input type="submit" value="GET EARLY ACCESS" id="button"></div>
+     </div>
+         <P id="foot">User &copy; 2019<br>
+         Made with <img src="love.png"> in Poland<br>
+         We're using cookies to improve your experience.
+         </P>
```

Comment:
Yyy? To jest ble. Bardzo ble. Jeśli chcesz przejść do następnej linii to tworzysz p. Te `<br>` są tu potrzebne jak sanki w lato...